woocommerce/woocommerce
# Onboarding wizard cleanup

### Changes proposed in this Pull Request:

While working on #23690, we noticed that there was room for some cleanup in the onboarding wizard code that decides which payment gateways to display to the user. We opt to leave this work to a subsequent PR since #23690 was included in a patch release. This PR removes two images that are not used anymore since https://github.com/woocommerce/woocommerce/pull/23690/commits/e131b01133cbac7a381322b18dd5a70a1345dfb9 (see https://github.com/woocommerce/woocommerce/pull/23690#discussion_r283994390) and removes some code duplication in WC_Admin_Setup_Wizard::get_wizard_in_cart_payment_gateways() (see https://github.com/woocommerce/woocommerce/pull/23690#discussion_r284004924).

### How to test the changes in this Pull Request:

1. Check that assets/images/klarna-white.png and assets/images/square-white.png are indeed not used anymore
2. Test that the behavior of the second step of the onboarding wizard, where users can enable payment gateways, remains the same. See p6q8Tx-VJ-p2 for more information on the expected behavior.

### Changelog entry

> Removes unused images assets/images/klarna-white.png and assets/images/square-white.png
